portability-api/src/main/java/org/datatransferproject/api/ApiMain.java

```diff
@@ -167,17 +166,22 @@ public void initializeHttps(
       throw new RuntimeException(e);
     }
 
-    Injector injector =
-        Guice.createInjector(
-            new ApiServicesModule(
-                typeManager,
-                cloudExtension.getJobStore(),
-                keyGenerator,
-                trustManagerFactory,
-                keyManagerFactory,
-                authServiceExtensions,
-                tokenManager,
-                extensionContext));
+    Injector injector = null;
+    try {
+      injector =
+          Guice.createInjector(
+              new ApiServicesModule(
+                  typeManager,
+                  cloudExtension.getJobStore(),
+                  keyGenerator,
+                  trustManagerFactory,
+                  keyManagerFactory,
+                  authServiceExtensions,
+                  tokenManager,
+                  extensionContext));
+    } catch (Exception e) {
+      monitor.info(() -> "Unable to initialize Guice in Apia", e);
```

Comment:
Also why not re-throw the exception as well?  Won't this lead to a NPE or some other hard to understand error latter?

Reply:
Done. Had it but lost it in copy and paste.